homeassistant/components/android_ip_webcam/switch.py

```diff
@@ -21,8 +22,8 @@
 class AndroidIPWebcamSwitchEntityDescriptionMixin:
     """Mixin for required keys."""
 
-    on_func: Callable[[PyDroidIPCam], None]
-    off_func: Callable[[PyDroidIPCam], None]
+    on_func: Callable[[PyDroidIPCam], Coroutine[Any, Any, bool]]
+    off_func: Callable[[PyDroidIPCam], Coroutine[Any, Any, bool]]
```

Comment:
It it returns None, then it's not possible to await it...
So the type hint was incorrect here.